README.md

```diff
@@ -384,6 +384,29 @@ Container images used in the tests are:
   - version 10.3: `registry.redhat.io/rhscl/mariadb-103-rhel7` (only if `ts.authenticated-registry` is set)
 - MSSQL: `mcr.microsoft.com/mssql/rhel/server`
 
+### `infinispan-client`
+
+Verifies the way of sharing cache with Datagrid operator.
+
+#### Prerequisities
+- Datagrid operator installed in `datagrid-operator` namespace. This needs cluster-admin rights to install.
+- The operator supports only single-namespace so it has to watch another well-known namespace `datagrid-cluster`. 
+This namespace must be created by "qe" user or this user must have access to it because infinispan tests are connecting to it.
+- These namespaces should be prepared after the Openshift installation.
```

Comment:
I think it would be really useful to have a link to operator installation docs, and to the specific chapter - the GUI install does not really allow to specify the `datagrid-cluster` (or at least I have not found a reasonable way)